docs/overview/community/ComponentGuidelines.stories.mdx

```diff
@@ -0,0 +1,178 @@
+import { Title, Meta } from "@storybook/addon-docs";
+
+<Meta title="Overview/Community/Component Guidelines" />
+
+<Title>Component Guidelines</Title>
+
+This guide aims to provide insights and recommendations for building and contributing
+robust and adaptable components that are consistent with the existing UI Kit library.
+
+## File Structure & Purpose
+
+The directory should contain the following files:
+
+```sh
+MyComp/
+├── index.ts            # entry file with the component's public exports
+├── MyComp.tsx          # component implementation and types
+├── MyComp.stories.tsx  # component documentation and examples
+├── MyComp.styles.ts    # component classes and stying utils
+└── MyComp.test.tsx     # component unit tests
+```
+
+## Guidelines
```

Comment:
Maybe we should have a bullet point saying that we also need to export the component on the `index.ts` file of the components folder?

Reply:
Isn't that the
> Components MUST be exported from the `index.ts` file.

I can rephrase it